src/GeositeFramework/js/Export.js

```diff
@@ -1,12 +1,121 @@
+        }
+
+        function createUiContainer(view, paneNumber) {
+            var model = view.model,
+                $uiContainer = $($.trim(N.app.templates['template-export-window']({id: 'export-print-preview'})));
+
+            view.$uiContainer = $uiContainer;
+
+            var pluginCssPath = 'css/app-print.css',
+                printCssClass = 'plugin-print-css',
+                oppositePaneHideCssPath = 'css/print-hide-map' +
+                    (paneNumber === 0 ? 1 : 0) + '.css';
+
+
+            // Add the plugin css
+            addCss(pluginCssPath, printCssClass);
+
+            // Hide the possible same plugin from the opposite map pane
+            addCss(oppositePaneHideCssPath, printCssClass);
+
+            // Give some time for the browser to parse the new CSS,
+            // if this wasn't slightly delayed, occasionally the override would
+            // not be present and print features wouldn't show up.
+            _.delay(function() {}, 200);
+
+            // Attach to top pane element
+            view.$el.parents().find('.content .nav-apps').after($uiContainer);
+
+            // Tell the model about $uiContainer so it can pass it to the plugin object
+            model.set('$uiContainer', $uiContainer);
+        }
+
+        function setupPrintableMap() {
+            var mapMarkup = N.app.templates['template-export-window']({pluginName: "Test"}),
+                $mapPrint = $($.trim(mapMarkup)),
+                $printPreview = $('#print-preview-sandbox'),
+                mapReadyDeferred = $.Deferred(),
+                mapNodeParent = $("#map-0")[0].parentElement;
+
+            // Setup a print-preview window for the user to select an extent and zoom level
+            // that will be persisted at print due to its fixed size.
+            
+            TINY.box.show({
+                animate: false,
+                html: $mapPrint[0].outerHTML,
+                boxid: 'export-print-preview-container',
+                width: 605,
+                height: 625,
+                fixed: true,
+                maskopacity: 40,
+                openjs: function () {
+                    var mapNode = $("#map-0").detach()[0];
+                    $("#export-print-preview-map").append(mapNode);
+
+                    $('#export-button').on('click', function() {
+                        var $printSandbox = $('#map-print-sandbox'),
+                            parseDeferred = $.Deferred(),
+                            previewDeferred = $.Deferred();
+                        $('.print-sandbox-header h1').text($("#export-title").val());
+
+                        _.delay(parseDeferred.resolve, 500);
+
+                        // Move the map from the plugin print preview dialog to the sandbox where
+                        // the plugin can mess with it's positioning among its other elements
+                        //var mapReadyDeferred = this.printableMap($printSandbox, previewDeferred);
```

Comment:
Consider updating this comment to reflect the use of this code in print the main map and not a plugin. Also, can the commented out code line be removed?